uber/templates/mivs_admin/studios.html

```diff
@@ -26,16 +26,16 @@ <h3>
       <a href="../mivs_applications/continue_app?id={{ studio.id }}" target="_blank">{{ studio.name }}</a>
     </td>
     <td>
-      {%- if studio.primary_contact.matching_attendee -%}
-        <a href="../registration/form?id={{ studio.primary_contact.matching_attendee.id }}">{{ studio.primary_contact.full_name }}</a>
+      {%- if studio.primary_contacts[0].matching_attendee -%}
```

Comment:
Why do we still randomly select the first 'primary contact' here? Because that's what we do in the MITS plugin, that's why.